spring-cloud-stream-binders/spring-cloud-stream-binder-spi/src/main/java/org/springframework/cloud/stream/binder/BinderCleaner.java

```diff
@@ -13,19 +13,19 @@
  * limitations under the License.
  */
 
-package org.springframework.xd.dirt.integration.bus;
+package org.springframework.cloud.stream.binder;
 
 import java.util.List;
 import java.util.Map;
 
 
 /**
- * Interface for implementations that perform cleanup for message buses.
+ * Interface for implementations that perform cleanup for binders.
  *
  * @author Gary Russell
  * @since 1.2
  */
-public interface BusCleaner {
+public interface BinderCleaner {
```

Comment:
should this be named BindingCleaner since it actually cleans the bindings, not the binder?